polymetis/polymetis/python/polymetis/robot_interface.py

```diff
@@ -285,6 +303,10 @@ def __init__(
         self,
         time_to_go_default: float = 1.0,
         use_grav_comp: bool = True,
+        use_mirror_sim: bool = False,
+        mirror_cfg: DictConfig = None,
+        mirror_ip: str = "",
+        mirror_port: int = -1,
```

Comment:
In the current implementation, configuration of the mirror sim is spread across both the parent class (`BaseRobotInterface`) and the child class (`RobotInterface`) without an obvious logic to which part should be in which class.
I think it would be cleaner to either:
1. Set up all mirror sim member variables and methods in parent class (`BaseRobotInterface`). The child class will simply be calling the appropriate methods in `go_home` (and maybe `move_to_joint_positions` and `move_to_ee_pose`)
2. Set up all mirror sim member variables and methods in child class (`RobotInterface`). You would have to call `super().send_torch_policy` within `RobotInterface.send_torch_policy` in order to add mirror sim stuff to the inherited method.

Let me know if you have any other thoughts on the design, happy to iterate!